Cargo.toml

```diff
@@ -26,7 +26,8 @@ regex = "1.0"
 bufstream = "0.1.3"
 imap-proto = "0.16.1"
 nom = { version = "7.1.0", default-features = false }
-base64 = "0.13"
+rsasl = { version = "2.0.0-rc.2", default-features = false, features = ["std", "provider_base64"], optional = true }
+base64 = "0.13.0"
```

Comment:
I don't think you need to modify `base64` here now?

Reply:
Whoops that was by mistake ^^'